main.cpp

```diff
@@ -173,63 +175,65 @@ struct CarWash //2)
 
 
 /*
-1)
+1) Delay
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) time knob
+    2) feedback knob
+    3) repeat knob
+    4) spread knob
+    5) dry/wet knob
 3 things it can do:
-    1)
-    2)
-    3)
+    1) repeat the signal
+    2) feedback the signal
+    3) spread the signal
  */
 
+
 /*
-2)
+2) Filter
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) frequence knob
+    2) resonance knob
+    3) modulation knob
+    4) lfo knob
+    5) filter typ knob
 3 things it can do:
-    1)
-    2)
-    3)
+    1) cut out frequencies 
+    2) modulate frequencies
+    3) color the sound 
  */
 
+
 /*
-3)
+3) - Synthesizer
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) oscillator
+    2) adsr
+    3) filter
+    4) midi input / output
+    5) audio output
 3 things it can do:
-    1)
-    2)
-    3)
+    1) create sounds
+    2) send midi
+    3) manipulate the sound with adsr evelopes
  */
```

Comment:
You need to remember that you'll be turning these 5 properties into member variables.  Member variables whose type is a primitive, except for in the 10th UDT.  in the 10th UDT, your 5 properties must be formed from your 9 other types.   
This `Synthesizer` is a perfect example of what a 10th UDT should be.  your 3rd property `Filter` is already a UDT: `#2`
- [ ] replace `Synthesizer` with something else.  
- [ ] make `Synthesizer` your `#10` UDT for CheckPoint2

Reply:
- replaced Synthesizer with something else
- made Synthesizer your #10 UDT for CheckPoint2